src/WireMock.Net/WireMock.Net.csproj

```diff
@@ -3,7 +3,7 @@
     <Description>Lightweight Http Mocking Server for .Net, inspired by WireMock from the Java landscape.</Description>
     <AssemblyTitle>WireMock.Net</AssemblyTitle>
     <Authors>Stef Heyenrath</Authors>
-    <TargetFrameworks>net451;net452;net46;net461;netstandard1.3;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net451;net452;net46;net461;netstandard1.3;netstandard2.0;netstandard2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
netcoreapp3.0 is end of life, so no need to add this.

https://dotnet.microsoft.com/platform/support/policy/dotnet-core